backend/templates/model-na-31-2.typ

```diff
@@ -0,0 +1,351 @@
+enerzijds het aantal toegelaten kiezers zoals vastgesteld door het stembureau en anderzijds het aantal uitgebrachte
+stemmen zoals vastgesteld door het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] (zie rubriek 3 van
+bijlage(n) 2 bij dit proces-verbaal).
+
+= Bij gecombineerde stemmingen
+
+Zijn de verzegelde pakken van een andere verkiezing geopend omdat er een gegrond vermoeden bestond dat stembiljetten,
+stempassen, volmachtbewijzen of kiezerspassen van de eigen verkiezing zich in deze pakken bevonden?
+
+#block_with_checkbox(checked: false)[
+  Nee
+]
+
+#block_with_checkbox(checked: false)[
+  Ja, er zijn verzegelde pakken van de verkiezing van #text_input(value: none, empty_width: 120pt) geopend.
+  Hierin zijn aangetroffen en bij de stemopneming van de eigen verkiezing betrokken:
+
+  #emph[Noteer hier het nummer van het betreffende stembureau en het aantal stembiljetten, dan wel stempassen, volmachtbewijzen of kiezerspassen dat is aangetroffen.]
+
+  #alertbox(height: 200pt)[]
+]
+
+#pagebreak(weak: true)
+
+= Bezwaren van kiezers tijdens de zitting van het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam]
+
+Bezwaren zijn klachten of aanmerkingen van kiezers. Voorbeelden van bezwaren van kiezers: een stem op een stembiljet
+wordt onterecht geldig, ongeldig of blanco verklaard, het resultaat van de telling wordt niet bekend gemaakt, het resultaat van
+de telling is onjuist.
+
+#emph[
+  Leg alle bezwaren van kiezers die zijn ingebracht tijdens de zitting van het
+  #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] vast in dit proces-verbaal. Ook als het
+  #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] het er niet mee eens is. Heeft het
+  #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] een reactie op het bezwaar? Noteer die dan ook. #underline[*Let op:*] noteer geen persoonsgegevens van kiezers.
+]
+
+#alertbox(height: 200pt)[]
+
+= Onregelmatigheden of bijzonderheden
+
+#emph[
+  Omschrijf de onregelmatigheden of bijzonderheden tijdens de zitting en vermeld hoe vaak hier sprake van was.
+  Denk onder andere ook aan de situatie dat er onverhoopt iets is misgegaan tijdens het vervoer en/of de opslag
+  van de stembescheiden. #underline[*Let op:*] noteer geen persoonsgegevens van kiezers.
+]
+
+#alertbox(height: 200pt)[]
+
+#pagebreak(weak: true)
+
+= Aantal stemmen per lijst en per kandidaat in #is_municipality[de gemeente][het openbaar lichaam]
+
+#for political_group in input.summary.political_group_votes [
+  #table(
+    columns: (80pt, 1fr, auto),
+    inset: 8pt,
+    stroke: 0.5pt + black,
+    fill: (_, y) => if y > 2 and calc.even(y) { rgb("EAF2F5") },
+    table.header(
+      table.cell(colspan: 3, grid(
+        columns: (auto, auto),
+        gutter: 12pt,
+        [*Lijstnaam*],   [#TODO],
```

Comment:
Note that list name and candidate names are currently unavailable, which does result in a large set of TODOs right now, is there something else we could display here instead?

Reply:
Why are they unavailable?

Reply:
I must be missing something, where would they be stored?